Common subdirectories: drunken-bishop-0.1.0.0/Data and drunken-bishop-0.1.0.0.fixed/Data
diff -u -N drunken-bishop-0.1.0.0/drunken-bishop.cabal drunken-bishop-0.1.0.0.fixed/drunken-bishop.cabal
--- drunken-bishop-0.1.0.0/drunken-bishop.cabal	2001-09-09 03:46:40.000000000 +0200
+++ drunken-bishop-0.1.0.0.fixed/drunken-bishop.cabal	2024-07-06 14:01:52.425566252 +0200
@@ -25,7 +25,7 @@
   ghc-options:         -Wall
   build-depends:       base >=4.7 && <5
                      , array >= 0.5.4 && <0.6
-                     , bytestring >= 0.10.10 && <0.11
+                     , bytestring >= 0.10.10 && <0.12
                      , pureMD5 >= 2.1.3 && <2.2
   default-language:    Haskell2010

@@ -41,6 +41,6 @@
   ghc-options:         -Wall
   build-depends:       base >=4.7 && <5
                      , drunken-bishop
-                     , bytestring >= 0.10.10 && <0.11
+                     , bytestring >= 0.10.10 && <0.12
                      , cryptohash-sha256 >= 0.11.101 && <0.12
   default-language:    Haskell2010
diff -u -N drunken-bishop-0.1.0.0/Setup.hs drunken-bishop-0.1.0.0.fixed/Setup.hs
--- drunken-bishop-0.1.0.0/Setup.hs	1970-01-01 01:00:00.000000000 +0100
+++ drunken-bishop-0.1.0.0.fixed/Setup.hs	2024-07-06 13:47:20.165305997 +0200
@@ -0,0 +1,3 @@
+import Distribution.Simple
+main = defaultMain
+
Common subdirectories: drunken-bishop-0.1.0.0/src and drunken-bishop-0.1.0.0.fixed/src
